website/content/sdk/guides/intros/stocks/stocks-screener.md

```diff
@@ -0,0 +1,314 @@
+|  6 | ED       |   0.32 |  1.83 |  0.0574 |  0.0843 |   0.0365 |    -0.0578 |    0.2426 | 66.88 |   96.3  |  -0.0025 |      0.002  | -0.0045 |      1.3419e+06  |
+|  7 | SFM      |   0.41 |  1.12 |  0.0584 |  0.1446 |   0.1605 |    -0.0518 |    0.4854 | 63.99 |   33.51 |  -0.0095 |     -0.0053 | -0.0041 |      1.14442e+06 |
+|  8 | PINC     |   0.36 |  0.7  |  0.0322 | -0.0004 |  -0.0711 |    -0.2042 |    0.0773 | 55.19 |   33.2  |   0.0097 |      0.0125 | -0.0027 | 373171           |
+|  9 | AEP      |   0.41 |  2.07 |  0.0456 |  0.0559 |  -0.0117 |    -0.1037 |    0.1799 | 61.62 |   94.65 |  -0.006  |      0.0005 | -0.0065 |      2.04444e+06 |
+| 10 | LRN      |   0.25 |  1.36 |  0.0302 | -0.0923 |  -0.0562 |    -0.2439 |    0.3957 | 44.85 |   35.8  |  -0.0047 |     -0.0014 | -0.0033 | 270842           |
+| 11 | AMN      |   0.3  |  4.85 |  0.011  |  0.0599 |   0.1431 |    -0.0551 |    0.4744 | 54.13 |  122.01 |  -0.0078 |     -0.0059 | -0.0019 | 264138           |
+
+### Valuation
+
+Lastly, `valuation` gets the basic fundamental ratios.
+
+|    | Ticker   |   Price |   Change |   Volume |   Market Cap |   P/E |   Fwd P/E |   PEG |   P/S |   P/B | P/C   | P/FCF   |   EPS this Y |   EPS next Y |   EPS past 5Y |   EPS next 5Y | Sales past 5Y   |
+|---:|:---------|--------:|---------:|---------:|-------------:|------:|----------:|------:|------:|------:|:------|:--------|-------------:|-------------:|--------------:|--------------:|:----------------|
+|  0 | AAPL     |  141.17 |  -0.0211 | 83578852 |  2.23252e+12 | 23.14 |     20.74 |  2.6  |  5.66 | 44.67 | 46.22 | 23.11   |        0.089 |       0.0898 |        0.216  |        0.0889 | 0.115           |
+|  1 | MSFT     |  240.33 |  -0.0059 | 17929705 |  1.77308e+12 | 25.9  |     21.49 |  1.99 |  8.73 | 10.32 | 16.53 | 39.59   |        0.198 |       0.1709 |        0.243  |        0.1301 | 0.155           |
+|  2 | GOOG     |   95.44 |  -0.0084 | 20145081 |  1.23434e+12 | 18.68 |     18.04 |  2.09 |  4.38 |  4.88 |  N/A  |  N/A    |       -0.159 |       0.121  |        0.2848 |        0.0895 |  N/A            |
+|  3 | GOOGL    |   95.19 |  -0.009  | 20047848 |  1.21605e+12 | 19.51 |     18.12 |  2.18 |  4.31 |  4.89 | 10.46 | 19.44   |        0.914 |       0.1139 |        0.321  |        0.0895 | 0.233           |
+|  4 | AMZN     |   92.42 |  -0.0163 | 65245723 |  9.3352e+11  | 85.26 |     55.08 |  3.28 |  1.86 |  6.85 | 15.91 |  N/A    |        0.549 |       0.8239 |        0.676  |        0.26   | 0.281           |
+
+Combined, the five DataFrames provide an outline of who they are, which segment of they market they belong to, and how they are currently trading.
+
+### Unconventional Applications
+
+One way to use the screener is to feed it the least amount of variables possible. In this next example, the preset file contains only one filter; requesting data for all constituents of the S&P 500 Index.
+
+```console
+# Author of preset: Danglewood
```

Comment:
Please change this to OpenBB.

Reply:
Done and updated in the attached ZIP file